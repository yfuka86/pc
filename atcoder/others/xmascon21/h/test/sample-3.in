8 7
0 135281516 667138242 534705053 894609006 363845551 983263711 368399563
706521153 0 205503439 17581194 48971395 248346723 723069160 809331769
95374102 177083480 0 966474089 652931117 138712346 184755987 158919475
378114185 853217671 858087623 0 385823507 528768155 637814125 154972838
19176284 943976794 592288689 814797836 0 36241010 768079008 314765803
951276099 885340801 394097671 282385830 497465585 0 69878345 741774067
106863220 613773019 880926179 252272587 687103226 598098347 0 464666892
209619802 423789973 917284938 456850501 156569668 254002916 550388735 0
0 374063803 312003894 140142446 439797362 925129489 698118121
224246820 0 481191147 684671827 698236806 587586359 495418696
389630714 220172615 0 865126951 43296181 113977262 779586506
686309670 840140432 20859844 0 68039199 899714832 282973414
779591352 625690436 674759197 529126191 0 797360620 123804494
621741551 994747737 802674384 746120527 131201061 0 390573239
715533801 898645859 794564126 102532744 421471382 705169995 0
