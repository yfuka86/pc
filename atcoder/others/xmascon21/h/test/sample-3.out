131557205 340211781 754428084 483092626 214955141 879575978 855361045
154851590 609864261 593925945 419466834 304837127 133113507 736477090
54632615 194927386 72101830 731318386 879946775 993433194 425542545
482048637 5880376 457649402 761233161 514923047 377996184 873772583
675269156 634883332 185904773 77524684 888694113 590920163 138406866
935188259 349594389 150776180 792068517 879123091 904461829 898840329
950638863 89008070 933320378 534103502 190891412 830891397 981135229
848888881 417544354 637718293 793237398 440680172 52239789 963945205
